pkg/server/proto/server.proto

```diff
@@ -458,6 +458,18 @@ service Waypoint {
   // project config to the Waypoint database.
   rpc ConfigSyncPipeline(ConfigSyncPipelineRequest) returns (ConfigSyncPipelineResponse);
 
+  //----------------------------------------------------------------------
+  // ProjectTemplate endpoints
+  //----------------------------------------------------------------------
+
+  rpc UpsertProjectTemplate(UpsertProjectTemplateRequest) returns (UpsertProjectTemplateResponse);
```

Comment:
Following the convention, let's add some comments! Those will turn into swagger api docs.

Reply:
Added!